the-collab-lab/tcl-34-smart-shopping-list
# Delete products

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

In this PR, we allow the user to delete products from their list. A confirmation popup is presented to the user before deleting the Firebase product, preventing the user from unintentionally deleting products.

## Related Issue

Closes #11 

## Acceptance Criteria

- [x] The user can remove an item from the shopping list.
- [x] Before deleting, ask the user to confirm that they really want to delete the item to avoid accidental deletions.
- [x] The deletion should cause the associated records in the database to be deleted.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

![delete](https://user-images.githubusercontent.com/32148199/141004953-5cdea23e-da8d-47e2-93aa-cb5d8da658a6.gif)

## Testing Steps / QA Criteria

1. From your terminal, pull down this branch with `git pull origin is-mm-delete-items` and check that branch out with `git checkout is-mm-delete-items`
2. Then `npm start` to launch the app
3. In the Shared Token field enter the `isa-collection` token
4. Try to delete a product
